EIPS/eip-7577.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: 7577
+title: Versioning Scheme for EIPs
+description: Use a versioning scheme for EIPs based on changes made to their Specification section.
+author: danceratopz (@danceratopz), Ahmad Bitar (@smartprogrammer93)
+discussions-to: https://ethereum-magicians.org/t/add-eip-versioning-scheme-for-eips/17295
+status: Draft
+type: Meta
+created: 2023-12-13
+---
+
+## Abstract
+
+This EIP introduces a versioning scheme for [Standards Track](./eip-1.md#eip-types) EIPs by applying semantic versioning based on changes made to the EIP's Specification section once its status has changed from `Draft` to `Review`.
+
+## Motivation
+
+EIP specifications often receive increasing modifications as more people review them, which is generally the case as client teams start implementing the specifications and the community gains a better understanding of their interaction with the rest of the protocol. These changes can be difficult to track. In particular, as EVM reference tests are often not maintained (and generally not released) by client teams or the EIP's authors, it can be difficult to ascertain whether a release of reference tests (for example from ethereum/tests or ethereum/execution-spec-tests) is sufficient, or even valid, to test the latest version of an EIP's specifications or the specification as currently implemented by a client.
```

Comment:
```suggestion
EIP specifications often receive increasing modifications as more people review them, which is generally the case as client teams start implementing the specifications and the community gains a better understanding of their interaction with the rest of the protocol. These changes can be difficult to track. In particular, as EVM reference tests are often not maintained (and generally not released) by client teams or the EIP's authors, it can be difficult to ascertain whether a release of reference tests is sufficient, or even valid, to test the latest version of an EIP's specifications or the specification as currently implemented by a client.
```

Avoid using external links.